pkg/test/framework/components/istio/ingress.go

```diff
@@ -37,8 +37,8 @@ const (
 	defaultIngressIstioLabel  = "ingressgateway"
 	defaultIngressServiceName = "istio-" + defaultIngressIstioLabel
 
-	eastWestIngressIstioLabel  = "eastwestgateway"
-	eastWestIngressServiceName = "istio-" + eastWestIngressIstioLabel
+	EastWestIngressIstioLabel  = "eastwestgateway"
```

Comment:
Is this used outside of the package?

Reply:
Oh yeah I had exposed it to do something that ended up being reverted. Thanks!